api/lib/QueryHandler.js

```diff
@@ -74,9 +74,22 @@ let mergeAddresses = function(addresses) {
   });
 };
 
+const sortMergedTenancies = merged => {
+  sorted_tenancies = merged.tenancies.sort(compareDateStrings);
+  merged.tenancies = { current: [], previous: [] };
+  sorted_tenancies.map(t => {
+    if (t.endDate === null && merged.tenancies.current.length === 0) {
+      merged.tenancies.current.push(t);
+    } else {
+      merged.tenancies.previous.push(t);
+    }
+  });
+};
+
 // Merge and tidy response upjects from multiple backends
 let mergeResponses = function(responses) {
   let merged = merge(...responses);
+  if (merged.tenancies) sortMergedTenancies(merged);
```

Comment:
Assign using the returned value:

if (merged.tenancies) merged.tenancies = sortMergedTenancies(merged);